lib/index.js

```diff
@@ -200,3 +189,25 @@ async function listenHttps() {
 async function listen() {
   await Promise.all([listenHttp(), listenHttps()]);
 }
+
+function _parseCerts({caFiles}) {
+  const parsedCerts = [];
+  if(typeof caFiles === 'string' ||
+  (Array.isArray(caFiles) && caFiles.length > 0)) {
+    if(!Array.isArray(caFiles)) {
+      caFiles = [caFiles];
+    }
+    // all certs must be parsed individually
+    caFiles.forEach(file => {
+      let bundle = fs.readFileSync(file);
+      try {
+        const certs = forge.pem.decode(bundle);
+        bundle = certs.map(cert => forge.pem.encode(cert));
+      } catch(e) {
+        throw new Error(e.message);
+      }
+      parsedCerts.push.apply(parsedCerts, bundle);
+    });
+  }
+  return parsedCerts;
```

Comment:
Based on the above comment, I have some suggestions for how to refactor this code.

Additionally, looking at this (very) old code, with fresh eyes, I'm wondering if it's necessary any more to do this? I suspect this code was applicable to a very old version of node where, if the bundle had a problem or if you had more than one cert in a given file, node did not handle it properly. We're talking like node 0.6.x or something ancient. Looking at the new docs, it seems that maybe we don't need to do all this work anymore:

> Any string or Buffer can contain multiple PEM CAs concatenated together.

https://nodejs.org/api/tls.html#tlscreatesecurecontextoptions

Given that, I'd like us to remove the PEM parsing entirely and just do this (again, I also shifted things around based on the first comment):

```suggestion
  if(!(typeof caFiles === 'string' ||
    (Array.isArray(caFiles) && caFiles.length > 0))) {
    // no certs to parse
    return;
  }
  
  const bundles = [];
  if(!Array.isArray(caFiles)) {
    caFiles = [caFiles];
  }
  // populate the bundles array from the file contents, noting that
  // node does accept multiple concatenated certs in a single file
  for(const file of caFiles) {
    const bundle = fs.readFileSync(file);
    bundles.push(bundle);
  }
  return bundles;
```

If the above works, we may be able to eliminate the dependency on node-forge as well.

Reply:
Ideally, we'd also use `fs` promises ... and read the files in parallel w/ a `Promise.all()`.